### PR TITLE
Rename CMAKE_ENABLE_CODE_COVERAGE to CLR_CMAKE_ENABLE_CODE_COVERAGE

### PR DESCRIPTION
**Pull request by omajid**
This was requested as part of https://github.com/dotnet/coreclr/pull/12039

cc @janvorli

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -394,7 +394,7 @@ isMSBuildOnNETCoreSupported()
                     __isMSBuildOnNETCoreSupported=1
                     ;;
                 "fedora.25-x64")
-                    __isMSBuildOnNETCoreSupported=1
+                    __isMSBuildOnNETCoreSupported=0
```
Comment (janvorli) [inline build.sh]: It seems this change was not intended, right?

Comment (omajid) [inline build.sh]: Doh. I cant build the managed stuff on my os. This was a workaround. Will fix.

Comment (janvorli) [inline build.sh]: Btw, there is a pending PR to fix that:
https://github.com/dotnet/coreclr/pull/12102

**Comment by omajid** [comment]
Thanks for including this fix!

**Pull request merged**